config/webpack/webpack.config.js

```diff
@@ -65,6 +65,38 @@ const makeCssLoaders = (options = {}) => {
   ];
 };
 
+const makeCssInJsLoaders = (options = {}) => {
+  const { server = false } = options;
+
+  return [
+    {
+      loader: require.resolve(`css-loader${server ? '/locals' : ''}`),
+      options: {
+        modules: true,
+        importLoaders: 3
+      }
+    },
+    {
+      loader: require.resolve('postcss-loader'),
+      options: {
+        plugins: () => [
+          require('autoprefixer')({
+            browsers: [
+              '> 1%',
+              'Last 2 versions',
+              'ie >= 10',
+              'Safari >= 8',
+              'iOS >= 8'
+            ]
+          })
+        ]
+      }
+    },
+    { loader: require.resolve('css-in-js-loader') },
+    { loader: require.resolve('babel-loader') }
```

Comment:
It's probably a good idea to use the sku Babel config here. That way the style files will have access to the module resolution and niceties like spreads.